common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -806,6 +806,29 @@ def test_perform_problem_reset_and_navigate(self):
         problem1_content_after_coming_back = self.problem_page.problem_content
         self.assertEqual(problem1_content_before_switch, problem1_content_after_coming_back)
 
+    def test_perform_problem_reset_and_focus(self):
+        """
+        Scenario:
+        I go to sequential position 1
+        Facing problem1, I select 'choice_1'
+        Then perform the action reset
+        Observe that the focus has shifted to the expected element
+        """
+        # Go to sequential position 1 and assert that we are on problem 1.
+        self.go_to_tab_and_assert_problem(1, self.problem1_name)
+
+        # Update problem 1's content state – by performing reset operation.
+        self.problem_page.click_choice('choice_choice_1')
+        self.problem_page.click_check()
+        self.problem_page.wait_for_expected_status('label.choicegroup_incorrect', 'incorrect')
+
+        # Reset and observe focus shift
+        self.problem_page.click_reset()
+        self.assertTrue(self.problem_page.is_focus_on_problem_meta())
+
+        # Check to ensure problem was reset
+        self.problem_page.wait_for_expected_status('span.unanswered', 'unanswered')
```

Comment:
Only use class names instead of depending on the HTML tag structure. Also if you are going to test that the unanswered state is "updated", verify it doesn't exist after you click check (so there is a change in state).